packages/apt/test.sh

```diff
@@ -147,6 +149,15 @@ echo "::endgroup::"
 
 run_test
 
+echo "::group::Test primary-maintainer"
+
+$(pg_config --bindir)/createuser postgres --superuser
+/host/packages/test-primary-maintainer.sh
+$(pg_config --bindir)/dropuser postgres
+
+echo "::endgroup::"
```

Comment:
Do we have to run this test after upgrading like executing `run_test` ?

Reply:
It should not be affected by the upgrade for now, but I moved it to `run_test` to run it just in case.